src/HeuristicCompletion/CoASTNodeFetcherHeuristic.class.st

```diff
@@ -0,0 +1,72 @@
+"
+I implement a fetcher heuristic for an AST node.
+My responsibility is to build a fetcher for a given node in a given context.
+I am implemented as a chain of responsibility: I treat the build request if possible, otherwise I forward the request to the next heuristic.
```

Comment:
```suggestion
I am implemented as a chain of responsibility: I treat the build request if possible, otherwise, I forward the request to the next heuristic.
```